dashboards/resource_dashboard.go

```diff
@@ -76,7 +78,17 @@ func ResourceDashboard() common.Resource {
 			newDashboardRequest.SerializedDashboard = content
 			createdDashboard, err := w.Lakeview.Create(ctx, newDashboardRequest)
 			if err != nil {
-				return err
+				if isParentDoesntExistError(err) {
```

Comment:
To remove a layer of nesting, we can do:
```go
if err != nil && isParentDoesntExistError(err)
```
since the else case of the outer condition is empty.

Reply:
thanks - fixed!